frappe/__init__.py

```diff
@@ -1636,15 +1643,18 @@ def append_hook(target, key, value):
 		target[key].extend(value)
 
 
-def setup_module_map():
-	"""Rebuild map of all modules (internal)."""
+def setup_module_map(site=None):
```

Comment:
this should always be site specific, because it's anyway only affecting site-specific cache. I don't even see any use-case where we need to allow passing site parameter, we could always use `frappe.local.site`.